### PR TITLE
chore: fix doc

### PR DESCRIPTION
**Pull request by 6293**

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -45,7 +45,7 @@ pub struct Baid58<const LEN: usize> {
 impl<const LEN: usize> Baid58<LEN> {
     /// # Panics
     ///
-    /// If HRI static string is longer than
+    /// If HRI static string is longer than [HRI_MAX_LEN]
     pub fn with(hri: &'static str, payload: [u8; LEN]) -> Self {
         debug_assert!(hri.len() <= HRI_MAX_LEN, "HRI is too long");
```
Comment (6293) [inline src/lib.rs]:
```rust
debug_assert!(hri.len() <= HRI_MAX_LEN, "HRI is too long");
```

why we disable this check in release build by using `debug_assert`?

File: src/lib.rs [hunk 2]
```diff
@@ -45,7 +45,7 @@ pub struct Baid58<const LEN: usize> {
 impl<const LEN: usize> Baid58<LEN> {
     /// # Panics
     ///
-    /// If HRI static string is longer than
+    /// If HRI static string is longer than [HRI_MAX_LEN]
```
Comment (dr-orlovsky) [inline src/lib.rs]:
```suggestion
    /// If HRI static string is longer than [`HRI_MAX_LEN`]
```

File: src/lib.rs (hunk 1 above)
Comment (dr-orlovsky) [reply src/lib.rs]: Since the assumption that the bug will be detected during the debugging. But you are right it is better to have just an assert

**Review by dr-orlovsky — APPROVED** [review]
ACK https://github.com/UBIDECO/rust-baid58/commit/13ef7f202d40fc4a67ad64393a6b266a6b8619da

**Pull request merged**